oqsprov/oqsprov.c

```diff
@@ -121,6 +121,50 @@ int oqs_patch_oids(void) {
     return 1;
 }
 
+const char* oqs_alg_encoding_list[OQS_OID_CNT/2] = { 0 };
+
+
+int oqs_patch_encodings(void) {
+///// OQS_TEMPLATE_FRAGMENT_ENCODING_PATCHING_START
```

Comment:
Should this not be #ifdef'd (dependent on USE_ENCODING_LIB)?

Reply:
Right, this is unused if USE_ENCODING_LIB is not set. Updated with ifdefs.